src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/jira/StandaloneStoppedNode.kt

```diff
@@ -137,6 +144,7 @@ internal data class StandaloneStoppedNode(
             if (deadline < now()) {
                 throw Exception("$uri failed to get out of $statusQuo status within $timeout")
             }
+            threadDump.gather(ssh, "thread-dumps")
```

Comment:
The commit doesn't explain why we're gathering threads during startup, but my guess is it's in order to diagnose what took so long. If that's so, if a startup phase is slow and times out, we won't have the thread dumps from that phase, because we `gather` after `throw Exception("$uri failed to get out of $statusQuo status within $timeout")`

Reply:
We're not going to have the last thread dump from the phase.

Reply:
Do we want to avoid the last thread dump?

Reply:
No. The intention was to gather the dumps before the deadline, but IMO it doesn't matter and we can gather one extra log.

Reply:
Since it was the intention, I'm with with both ways